### PR TITLE
Added a button to toggle fast decon mode

### PR DESCRIPTION
**Pull request by oof2win2**
Added a button in the top left bar to toggle between fast decon mode and normal decon mode. Requires #217 to be merged to function properly and show the button to only those who have the `fast-tree-decon` permission.

### CODE REVIEW

File: modules/addons/tree-decon.lua [hunk 1]
```diff
@@ -13,6 +15,22 @@ Global.register({ tree_queue, chache }, function(tbl)
...
+HasEnabledDecon:set_default(false)
+
+Gui.toolbar_button("entity/tree-01", {'tree-decon.main-tooltip'}, function (player)
+	return Roles.player_allowed(player, "fast-tree-decon")
+end)
+:on_click(function(player, element)
+	local status = HasEnabledDecon:get(player)
+	HasEnabledDecon:set(player, not status)
+	Gui.toolbar_button_style(element, not status)
+	player.print("Fast decon has been " .. (status and "disabled" or "enabled"))
```
Comment (Cooldude2606) [inline modules/addons/tree-decon.lua:30]: Change this into a locale and then I will merge this PR.

File: modules/addons/tree-decon.lua [hunk 2]
```diff
@@ -27,7 +27,7 @@ end)
 	local status = HasEnabledDecon:get(player)
 	HasEnabledDecon:set(player, not status)
 	Gui.toolbar_button_style(element, not status)
-	player.print("Fast decon has been " .. (status and "disabled" or "enabled"))
+	player.print({'tree-decon.toggle-msg'} .. " " .. (status and {'tree-decon.disabled'} or {'tree-decon.enabled'}))
```
Comment (Cooldude2606) [inline modules/addons/tree-decon.lua:30]: You must use locale concat here, eg `toggle-msg=Fast decon has been __1__` and `{'tree-decon.toggle-msg', {'tree-decon.disabled'}}`

Comment (oof2win2) [reply modules/addons/tree-decon.lua]: Oh, didn't know about that. ty!

**Comment by Cooldude2606** [comment]
Looks good to me.

**Pull request merged**